org.gnome.Tomboy.json

```diff
@@ -0,0 +1,225 @@
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/mono/dbus-sharp",
+                    /* the commit before unix FD passing, which is incompatible with Mono 4 */
+                    "commit": "f7dbfc28f2395e65bf6cf5961ba57aa2713e8715"
+                }
+            ]
+        },
+        {
+            "name": "dbus-sharp-glib",
+            "sources": [
+                {
+                    "type": "git",
+                    /* a branch from upstream to fix some build issues */
+                    "url": "https://github.com/hbons/dbus-sharp-glib",
+                    "commit": "9c003f566efa7cbcb413d6b374ccb15ec8abb330"
+                },
+                {
+                    "type": "patch",
+                    "path": "dbus-sharp-glib-no-system.patch"
+                }
+            ]
+        },
+        {
+            "name": "enchant",
+            "config-opts": ["--disable-static", "--with-myspell-dir=/usr/share/hunspell"],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                     "url": "https://github.com/AbiWord/enchant/releases/download/enchant-1-6-1/enchant-1.6.1.tar.gz",
+                     "sha256": "bef0d9c0fef2e4e8746956b68e4d6c6641f6b85bd2908d91731efb68eba9e3f5"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "autoreconf -vfi" ]
+                }
+            ]
+        },
+        {
+            "name": "gtkspell",
+            "config-opts": [
+                "--disable-gtk-doc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://http.debian.net/debian/pool/main/g/gtkspell/gtkspell_2.0.16.orig.tar.gz",
+                    "sha256": "8fc7dc560167b2cb7193e76aca625a152dc19b0ebf49816b78539cbb90d80d02"
+                }
+            ]
+        },
+        {
+            "name": "tomboy",
+            "config-opts": [ "--disable-schemas-install" ],
+            "post-install": [
+              "install -D -m644 org.gnome.Tomboy.appdata.xml /app/share/appdata/org.gnome.Tomboy.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ftp.gnome.org/pub/GNOME/sources/tomboy/1.15/tomboy-1.15.9.tar.xz",
+                    "sha256": "78d0a5f33db9b1077115773ad3291ecaf6e656f58545859b77a2ae7440abb248"
+                },
+                {
+                    "type": "file",
+                    "path": "org.gnome.Tomboy.appdata.xml"
+                },
+                {
+                    "type": "patch",
+                    "path": "tomboy-fix-mkdir.patch"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "autoreconf -vfi" ]
```

Comment:
Same comment about `rm-configure`, assuming the tarball has `autogen.sh`.

Reply:
Yeah also didn't work.